cookiecutter/bootstrap/cookiecutter.json

```diff
@@ -0,0 +1,12 @@
+{
+  "app_label": "noname",
```

Comment:
Why is app_label = "noname"?

Reply:
The bootstrap part is far from done...
But cookiecutter needs something there, and i wanted to prevent it being empty by default as not to overwrite the pulp-cli repositry accidentally.